src/modm/driver/radio/sx1276_definitions.hpp

```diff
@@ -0,0 +1,204 @@
+        Ocp = 0x0B,                 /// Overload current protection
+        Lna = 0x0C,                 /// Low Noise Amplifier Control
+        FifoAddrPtr = 0x0D,         /// Address Pointer in Fifo Buffer
+        FifoTxBaseAddr = 0x0E,      /// Base Write Address for Tx Fifo
+        FifoRxBaseAddr = 0x0F,      /// Base Read Address for Rx Fifo
+        FifoRxCurrentAddr = 0x10,   /// Fifo Address of the last received packet
+        IrqFlagsMask =  0x11,       /// Interrupt Masks
+        IrqFlags = 0x12,            /// Interrupt Flags
+        RxNbBytes = 0x13,           /// Amount of payload bytes in last received message
+        RxHeaderCntValueMsb = 0x14, /// MSB of valid header counter since last transition to rx mode
+        RxHeaderCntValueLsb = 0x15, /// LSB of valid header counter since last transition to rx mode
+        RxPacketCntValueMsb = 0x16, /// MSB of valid packet counter since last transition to rx mode
+        RxPacketCntValueLsb = 0x17, /// LSB of valid packet counter since last transition to rx mode
+        ModemStat = 0x18,           /// Modem status
+        PktSnrValue = 0x19,         /// Estimation of the signal-to-noise ration of the last packet received
+        PktRssiValue = 0x1A,        /// RSSI value of the last packet received
+        RssiValue = 0x1B,           /// Current RSSI value
+        HopChannel = 0x1C,          /// Configuration of the frequency hopping
+        ModemConfig1 = 0x1D,        /// Lora PHY configuration register 1
+        ModemConfig2 = 0x1E,        /// Lora PHY configuration register 2
+        SymbTimeoutLsb = 0x1F,      /// LSB symbol timeout (MSB is in ModemConfig 2 Register)
+        PreambleMsb = 0x20,         /// MSB of the preamble length
+        PreambleLsb = 0x21,         /// LSB of the preamble length
+        PayloadLength = 0x22,       /// Payload length
+        MaxPayloadLength = 0x23,    /// Maximum payload length
+        HopPeriod = 0x24,           /// Frequency Hopping Period
+        FifoRxByteAddr = 0x25,      /// Current Rx Fifo Pointer
+        ModemConfig3 = 0x26,        /// Lora PHY configuration register 3
+        PpmCorrection = 0x27,       /// Data rate offset control
+        FeiMsb = 0x28,              /// MSB of the estimated Frequency Error
+        FeiMid = 0x29,              /// Middle byte of the estimated Frequency error
+        FeiLsb = 0x2A,              /// LSB of the estimated frequency error
+        RssiWideband = 0x2C,        /// Wideband RSSI measurement
+        DetectOptimize = 0x31,      /// Lora Detection Optimize
+        InvertIQ = 0x33,            /// Invert I and Q
+        HighBWOptimize1 = 0x36,     /// High bandwidth optimization register 1
+        DetectionThreshold = 0x37,  /// Lora detection threshold
+        SyncWord = 0x39,            /// Lora syncword (0x34 for LoraWAN)
+        HighBWOptimize2 = 0x3A,     /// High bandwidth optimization 2
+        InvertIQ2 = 0x3B            /// Set to 0x19 for inverted IQ
```

Comment:
Does this work for documentation, or should this be like this?
```suggestion
        InvertIQ2 = 0x3B            ///< Set to 0x19 for inverted IQ
```